packages/slider/stories/slider.stories.ts

```diff
@@ -97,12 +145,234 @@ export const Disabled = (): TemplateResult => {
+                @change=${handleEvent}
+                ...=${spreadProps(args)}
+            >
+                <sp-slider-handle slot="handle" value="5"></sp-slider-handle slot="handle">
+            </sp-slider>
+        </div>
+    `;
+};
+
+export const TwoHandles = (args: StoryArgs): TemplateResult => {
+    const handleEvent = (event: Event): void => {
+        const target = event.target as SliderHandle;
+        if (target.value != null) {
+            if (typeof target.value === 'object') {
+                action(event.type)(target.value);
+            } else {
+                action(event.type)(`${target.name}: ${target.value}`);
+            }
+        }
+    };
+    return html`
+        <div style="width: 500px; margin: 12px 20px;">
+            <sp-slider
+                value="5"
+                step="1"
+                min="0"
+                max="255"
+                @input=${handleEvent}
+                @change=${handleEvent}
+                ...=${spreadProps(args)}
+            >
+                Output Levels
+                <sp-slider-handle slot="handle" name="min" value="5"></sp-slider-handle slot="handle">
+                <sp-slider-handle slot="handle" name="max" value="250"></sp-slider-handle slot="handle">
+            </sp-slider>
+        </div>
+    `;
+};
+TwoHandles.args = {
+    variant: 'range',
+    tickStep: 10,
+};
+
+export const ThreeHandlesOrdered = (args: StoryArgs): TemplateResult => {
+    const handleEvent = (event: Event): void => {
+        const target = event.target as SliderHandle;
+        if (target.value != null) {
+            if (typeof target.value === 'object') {
+                action(event.type)(target.value);
+            } else {
+                action(event.type)(`${target.name}: ${target.value}`);
+            }
+        }
+    };
+    return html`
+        <div style="width: 500px; margin: 12px 20px;">
+            <sp-slider
+                step="1"
+                min="0"
+                max="255"
+                label="Output Levels"
+                @input=${handleEvent}
+                @change=${handleEvent}
+                ...=${spreadProps(args)}
+            >
+                <sp-slider-handle slot="handle"
+                    name="low"
+                    value="5"
+                    max="next"
+                ></sp-slider-handle slot="handle">
+                <sp-slider-handle slot="handle"
+                    name="mid"
+                    value="100"
+                    min="previous"
+                    max="next"
+                ></sp-slider-handle slot="handle">
+                <sp-slider-handle slot="handle"
+                    name="high"
+                    value="250"
+                    min="previous"
+                ></sp-slider-handle slot="handle">
+            </sp-slider>
+        </div>
+    `;
+};
+ThreeHandlesOrdered.args = {
+    tickStep: 10,
+};
+
+// This is a very complex example from an actual application.
```

Comment:
Can you expand on how this work, I'll have no way of known if I break this in the future or not.

Also, what would you consider the "strangest" values this could produce? It might be good to set that as the default so that the VRT can support catching regressions here.

Reply:
It's quite cool that we can do this now, however!

Reply:
I am adding a long explanation. You can decide if it is worth keeping. I have changed the values so that they exercise the most complex part of the calculations by default.